.github/pull_request_template.md

```diff
@@ -13,3 +39,13 @@ things are done.
       added at least one reference to showcase the entry usage
       among our organization
 - [ ] I've described at least one use case for the entry
+
+<!-- END SECTION: CONTENT -->
+
+<!-- SECTION: CHORE -->
+### Motivations
+<!-- 
+Required. Add more context on why the proposed changes might be considered.
+-->
+
+<!-- END SECTION: CHORE -->
```

Comment:
ditto

Reply:
I'd avoid to put further burden on contributors keeping the template as the bare minum and focused on content writing.